_layouts/main.html

```diff
@@ -58,17 +58,19 @@ <h2>Meetings</h2>
 
 
  <div class="col-lg-4">
-  <h2>GSoC 2019: <br>Call for projects</h2>
+  <h2>GSoC 2019: <br> We got selected !</h2>
     <p>
-    <a href="/activities/gsoc.html"><img src="https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-icon-192.png"
+    <a href="https://indico.cern.ch/event/802397/"><img src="/images/gsocicon-meeting.png"
            alt="GSoC-logo" style="display: block; width:55%; margin-left: auto; margin-right: auto;"></a>
     </p>
     <p>
-    We would like to announce a <b>call for project proposals</b> for the <a href="/activities/gsoc.html">2019 HSF Google Summer of Code</a>.
+    CERN-HSF has been selected as a Google Summer of Code 2019 mentor organization.
     </p>
     <p>
-    If you are interested in submitting a project proposal, please do so by following <a href="/gsoc/guideline.html">these instructions</a>.
+    We will have an <a href="https://indico.cern.ch/event/802397/">informative meeting for mentors on <b>March 1st, at 14pm CET</b></a>.
```

Comment:
I would really not put a very specific meeting - it will just be out of date by Friday evening. Instead add an entry to the announcements/_posts directory with the meeting information (then Jekyll automatically expires it after Friday). There are many files there that show you how to do this.

Reply:
You are completely @graeme-a-stewart , thank you for the suggestion.  
I just modified it.